### PR TITLE
Handle podman divergence in cli commands, add checksum

### PR DESCRIPTION
**Pull request by copejon**
Add generated checksum file to release

### CODE REVIEW

**Comment by oglok** [comment]
If this PR is ready to be reviewed, can we change its name to be more descriptive? Like that, we'd have a better record of changes. Thanks

File: images/build/Dockerfile [hunk 1]
```diff
@@ -33,17 +33,14 @@ RUN make clean $MAKE_TARGET SOURCE_GIT_TAG=$SOURCE_GIT_TAG
...
-FROM registry.access.redhat.com/ubi8/ubi-minimal$DIGEST
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
 
 ARG ARCH=amd64
 
-RUN microdnf install -y \
-      policycoreutils-python-utils \
-      iptables && \
-      microdnf clean all
+RUN microdnf install -y policycoreutils-python-utils iptables &&  microdnf clean all
```
Comment (oglok) [inline images/build/Dockerfile:43]: Does the preview syntax create any kind of error?

File: scripts/release.sh [hunk 2]
```diff
@@ -203,35 +203,53 @@ build_container_images_artifacts() {
...
+}
+
+docker_create_manifest(){
+  local image_list="${*}"
+  local amend_images_options
+  for image in "${image_list[@]}"; do
+    amend_images_options+="--amend $image"
   done
+  podman manifest create "$IMAGE_REF_BASE:$VERSION" "${image_list[@]}" >&2
+}
```
Comment (oglok) [inline scripts/release.sh]: what is the difference between podman and docker images here?

Comment (copejon) [reply scripts/release.sh]: Podman and docker diverge slightly on the `manifest create` command.  `docker manifest create` takes a manifest name, and a list of other manifests.  You must add images by prefixing them with the `--amend` option.   Podman only allows you to pass one image to the `create` command inline. More images have to be added via `podman manifest add $IMAGE`. Docker does not have a `manifest add` command.

```
❯ docker manifest create --help

Usage:  docker manifest create MANIFEST_LIST MANIFEST [MANIFEST...]

Create a local manifest list for annotating and pushing to a registry

Options:
  -a, --amend      Amend an existing manifest list
      --insecure   Allow communication with an insecure registry
```

```
[root@fedora ~]# podman manifest create --help
Create manifest list or image index

Description:
  Creates manifest lists or image indexes.

Usage:
  podman manifest create [options] LIST [IMAGE]

Examples:
  podman manifest create mylist:v1.11
  podman manifest create mylist:v1.11 arch-specific-image-to-add
  podman manifest create --all mylist:v1.11 transport:tagged-image-to-add

Options:
      --all   add all of the lists' images if the images to add are lists

```

File: images/build/Dockerfile (hunk 1 above)
Comment (copejon) [reply images/build/Dockerfile:43]: Not sure what you mean by preview syntax.  This line doesn't produce errors for me on fed33/podman.

Comment (oglok) [reply images/build/Dockerfile:43]: sorry, I meant previous syntax. What's the purpose of changeing it to a oneliner?

File: scripts/release.sh (hunk 2 above)
Comment (oglok) [reply scripts/release.sh]: Awesome, thx!

File: images/build/Dockerfile (hunk 1 above)
Comment (copejon) [reply images/build/Dockerfile:43]: None other than to shorten the file - it wasn't the longest line in there.

**Pull request merged**